doc/en/source/https.rst

```diff
@@ -209,7 +209,7 @@ Terminology
 
```

Comment:
[Minor]--Security Architecture
1- both users are different better to give different name to them.
2- Rewrite the naming convention to segregate with each other-- write below suggestion name as discussed Generate user 1 token, Generate user 2 token, Device 1 token, Device 2 token, Device 3 token, Device 4 token
3- Write sequencing to show to flow

Reply:
1. Security Architecture png is updated.
2. Sequencing also done.